styles/style.css

```diff
@@ -0,0 +1,176 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.main {
+  display: flex;
+}
+
+h1 {
+  color: red;
```

Comment:
Remember to update colors before sending to Neal :)

Reply:
Will be certain to do this ;-)